FRE-nctools_documentation_github.txt

```diff
@@ -0,0 +1,273 @@
+   processors list. These information will be written into files, with file name is       
+   mask_table.n_mask.layout(1)Xlayout(2). The model is expected to read file mask_table   
+   if you want to mask out some processors.                                               
+                                                                                          
+   The ocean model is assumed to be logically rectangular grid and land model could be    
+   multiple tiles grid (like cubic shpere grid with 6 tiles). For land model, each        
+   processor list has three entries: layout position in x-direction, layout position in   
+   y-direction and tile number. For ocean model, each processor list has two entries:     
+   layout position in x-direction, layout position in y-direction.                        
+                                                                                          
+   The purpose of using mask domain is to decrease the processor usage without            
+   decreasing the performance, without changing bits relative to the case run without the 
+   mask domain option. The mask domain option acts to mask out regions that have zero     
+   ocean points (i.e., points that are all land) in both the ocean and sea ice models or  
+   mask out regions that have zero land points (i.e., points that are all ocean) in land  
+   model                                   
+
+2.Make_Remap_File
+
+  This program creates remap file from Cubic sphere grid to Cubic sphere grid.
+  The ratio of grid size of input and output grid must be integer. The purpose
+  of this tool is to create remap file on single processor for very high resoluton
+  grid ( It will large processor count and long time to create such file ). Currently
+  this tool can only remap file for data on grid cell center.
+
+3.Frigrid
+
+This program remaps (scalar or vector) data from the input grid to the output grid.
+
+4.Make_coupler_mosaic
+
+  This program Modifies land grid to match ocean grid at coast and calculate atmos/land,
+  atmos/ocean, and land/ocean overlaps using the Sutherland-Hodgeman polygon
+  clipping algorithm.
+
+5.Make_hgrid 
+
+  This program generates various types of horizontal grids in netCDF file format.
+
+6.Make_land_domain
+
+  This tools makes sure land_mask and land_ntile are consistent and pack the grid tile information into unstructured grid.
+
+7.Make_quick_mosaic
+
+  Make_quick_mosaic generates a complete grid a FMS coupler. When --ocean_topog  
+  is not specified, it takes a coupled mosaic as input_mosaic. Otherwise it     
+  takes a solo mosaic as input_mosaic. The atmosphere, ocean and land grid will 
+  be the same. The land/sea mask will be decided by the land/sea mask of        
+  input_mosaic if ocean_topog is not specified, otherwise by the ocean_topog    
+  file.  
+
+8.Make_topog
+
+  Make_topog can generate topography for any Mosaic. The output file                    

```

Comment:
Are you sure this works for any given mosaic? Also, I think it's important to say if this works with any input (observation/source) file or if the source file is built-in.